### PR TITLE
Feature/implement authentication on backend

### PR DESCRIPTION
**Pull request by viviviviviivilll**

### CODE REVIEW

File: BobManager.API/BobManager.API/BobManager.API.csproj [hunk 1]
```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
```
Comment (Sarhsian) [inline BobManager.API/BobManager.API/BobManager.API.csproj]: empty line

File: BobManager.API/BobManager.API/Controllers/Api/AccountController.cs [hunk 2]
```diff
@@ -39,21 +69,17 @@ private async Task<object> GenerateJwtToken(string email, User user)
...
             var expires = DateTime.Now.AddDays(Convert.ToDouble(_configuration["JwtExpireDays"]));
-
             var token = new JwtSecurityToken(
                 _configuration["JwtIssuer"],
                 _configuration["JwtIssuer"],
                 claims,
                 expires: expires,
                 signingCredentials: creds
             );
-
```
Comment (Sarhsian) [inline BobManager.API/BobManager.API/Controllers/Api/AccountController.cs:52]: empty line

**Review by Sarhsian — COMMENTED** [review]
Try to write code without empty lines.

**Review by Sarhsian — APPROVED** [review]
good

**Review by ander21 — APPROVED** [review]
Approve

**Review by Liosh-a — APPROVED** [review]
Approve

**Pull request closed without merging**